astor/rtrip.py

```diff
@@ -85,6 +85,10 @@
 dsttree = 'tmp_rtrip'
 
 
+def out_prep(s, pre_encoded=(type(b'') is type(''))):
```

Comment:
Yeah, it's basically a 2 vs 3 check, but in a duck-typing kind of way.  "I know I'm writing bytes to disk; are these the bytes I'm looking for?"

I'm open to other ways of doing it, but we have very few real 2 vs. 3 differences, so it doesn't seem to make sense to isolate them into their own module.

In any case, I'll certainly add a note to revisit if we drop legacy support.